force-app/main/default/classes/SOQL.cls

```diff
@@ -50,22 +52,32 @@ public with sharing class SOQL {
      * @return A list of Contact Ids.
      * Update the corresponding test class and testGetContactIdsByLastName method to cover this method
      */
-    //[public | private ] [static] data_type method_name (input parameters) {
-    //  The body of the method
-    //}
+    
+        public static List<Id> retrieveContactIds(String lastName1, String lastName2) {
+            
+           List<Id> contactIds = new List<Id>();
+            List<Contact> gatherContacts = [SELECT Id FROM Contact WHERE LastName = :LastName1 OR LastName = :LastName2 WITH SECURITY_ENFORCED];
+            for(Contact conId: gatherContacts){
+                contactIds.add(conId.Id);
+            }
+            return contactIds;
+    
+    }
```

Comment:
What happens if we have more than 2 LastName?

For more than 1 arg of the same type, you should use a collection like a Set or List.